### DISCUSSION TITLE
Trivy Dashboard in Headlamp

### DISCUSSION THREAD

**Discussion by mgalesloot** [aquasecurity/trivy#7720] [Ideas]
### Description

Headlamp is K8s dashboard and is extensible with plugins. https://headlamp.dev/

I have made a Headlamp plugin for the Trivy operator, supporting pages to view the Trivy custom resources. 

https://github.com/kubebeam/trivy-headlamp-plugin
https://artifacthub.io/packages/headlamp/trivy-headlamp/headlamp_trivy

The status is yet alpha.  Please have a look and share feedback. 

### Target

Kubernetes

### Scanner

None

**Comment by itaysk** [comment]
that's great to hear. please feel free to add your extension to the Trivy Ecosystem page https://github.com/aquasecurity/trivy/tree/main/docs/ecosystem

**Reply by mgalesloot** [reply]
PR for the ecosystem page: https://github.com/aquasecurity/trivy/pull/7916
